pkgs/development/compilers/flutter/engine/package.nix

```diff
@@ -0,0 +1,360 @@
+{ lib,
+  callPackage,
+  writeText,
+  symlinkJoin,
+  cacert,
+  targetPlatform,
+  hostPlatform,
+  darwin,
+  clang,
+  llvm,
+  tools ? callPackage ./tools.nix { inherit hostPlatform; },
+  stdenv,
+  stdenvNoCC,
+  runCommand,
+  fetchurl,
+  patchelf,
+  xorg,
+  libglvnd,
+  libepoxy,
+  wayland,
+  freetype,
+  pango,
+  glib,
+  harfbuzz,
+  cairo,
+  gdk-pixbuf,
+  at-spi2-atk,
+  zlib,
+  gtk3,
+  curl,
+  pkg-config,
+  ninja,
+  python3,
+  git,
+  version,
+  dartSdkVersion,
+  hashes,
+  patches,
+  url,
+  runtimeMode ? "release",
+  isUnoptimized ? false,
+}:
+with lib;
+let
+  expandSingleDep = dep: if lib.isDerivation dep then
+    ([ dep ] ++ map (output: dep.${output}) dep.outputs)
+  else [];
+
+  expandDeps = deps: flatten (map expandSingleDep deps);
+
+  constants = callPackage ./constants.nix {
+    inherit targetPlatform;
+  };
+
+  git-revision = writeText "git-revision.py" ''
```

Comment:
We should use writePythonScript instead

Reply:
Where is `writePythonScript`? I cannot find it in `nixpkgs` with `repl` and it isn't mentioned in the manual. I did a `grep -r writePythonScript` and found nothing.